src/data/nav-links.json

```diff
@@ -1,22 +1,2 @@
 [
-  {
```

Comment:
We could probably delete the whole file if we're not adding a navigation to the teaser page